Feat(data): Remnant End Of The Line (#245)
endless-sky/Endless-Sky-Delta: data/human/engines.txt

outfit "Flare color Ion medium"
	category "Engine Modifications"
	cost 35000
	thumbnail "outfit/blue screwdriver"
	"engine mod space" -0.1
	"mass" 0.1
	"acceleration multiplier" 0.05
	"flare sprite" "effect/plasma flare/medium"
		"frame rate" 1.0
	description "Making pretty colors in the sky is not the main focus of the Syndicate's production teams, nonetheless some manager dreamed up a way to let people pay for the privilege of making their exhaust glow in such a way as to appear to be using plasma thrusters. This version creates medium exhaust plumes."
	description "	Engine modifications can only be installed on an engine, and generally take up a number of connections that effectively limit any particular engine from having very many of them, regardless of its physical size."

outfit "Flare color Ion large"
	category "Engine Modifications"
	cost 45000
	thumbnail "outfit/blue screwdriver"
	"engine mod space" -0.1
	"mass" 0.1
	"acceleration multiplier" 0.05
	"flare sprite" "effect/plasma flare/large"
		"frame rate" 0.9

> 	description "Making pretty colors in the sky is not the main focus of the Syndicate's production teams, nonetheless some manager dreamed up a way to let people pay for the privilege of making their exhaust glow in such a way as to appear to be using plasma thrusters. This version creates large exhaust plumes."

Reviewer: Probably better to change this to:
```suggestion
	description "Making pretty colors in the sky is not the main focus of the Syndicate's production teams, but nonetheless some manager dreamed up a way to let people pay for the privilege of making their exhaust glow in such a way as to appear to be using plasma thrusters. This version creates large exhaust plumes."
```
(this probably goes for the rest of these descriptions as well)

Author: Thanks for pointing that out. I have pushed a fix now. (and credited you as co-authors on it)

	thumbnail "outfit/blue screwdriver"
	"engine mod space" -0.2
	"mass" 0.01
	"acceleration multiplier" 0.2
	"turn multiplier" 0.2
	"unique" 1
	description "James gave you this engine mod, along with a story about having been given it by some salvagers that he saved from pirates. He considers it a good luck charm, and noted that his ship always seemed to perform a little better when he had it installed."
	description "	This engine modification is hard to figure out: It has no seams, no obvious method of construction, doesn't match the styling of any known manufacturer, and doesn't quite feel like any material you know of. It does fit right onto your engine, though, and connects nicely with the regulator, power, and control systems."

outfit "Kraz Flight Controls"
	category "Engine Modifications"
	cost 50000
	thumbnail "outfit/blue screwdriver"
	"engine mod space" -0.2
	"mass" 5
	"turn multiplier" 0.1

> 	description "Kraz Cybernetics' flight control systems are the keystone of their corporate success. Virtually unknown outside of pilots and vehicle manufacturers, the basic flight control system of most starships is ultimately sourced from them. Building on that success, they created this after-market add-on for those interested in replacing the bare-bones flight control system that shipyards prefer with a more robust system. If this system is what is distributed on a galaxy-wide scale, it is possible they might have more advanced version in the company's home outfitter..."

Reviewer: ```suggestion
	description "Kraz Cybernetics' flight control systems are the keystone of their corporate success. Virtually unknown outside of pilots and vehicle manufacturers, the basic flight control system of most starships is ultimately sourced from them. Building on that success, they created this after-market add-on for those interested in replacing the bare-bones flight control system that shipyards prefer with a more robust system. If this system is what is distributed on a galaxy-wide scale, it is possible they might have a more advanced version in the company's home outfitter..."
```

outfit "Kraz Advanced Control Systems"
	category "Engine Modifications"
	cost 125000
	thumbnail "outfit/blue screwdriver"
	"engine mod space" -0.2
	"mass" 4
	"turn multiplier" 0.2
	description "Kraz Cybernetics' flight control systems are the keystone of their corporate success, and the baseline original equipment manufacturere versions are ubiquous in starships. Here in the poorly regulated fringe territories, Kraz's engineers have been able to dial up the performance and skirt the very edge of the safety margins in order to provide the ultimate in ship reaction times. How exactly they do this is unclear, but according to the packaging these changes are complimentary to those found in their other Kraz Flight Control outfit."
	description "	Engine modifications can only be installed on an engine, and generally take up a number of connections that effectively limit any particular engine from having very many of them, regardless of its physical size."

outfit "Brute Force Thrust mod"
	category "Engine Modifications"
	cost 125000
	thumbnail "outfit/blue screwdriver"
	"engine mod space" -0.2
	"mass" 10
	"acceleration multiplier" 0.2

> 	description "The engineer in front of you looks skeptically at your ship. 'Don't worry, this is a piece of cake. I can have her accelerating ten percent faster with a few hours of work. It'll add a bit 'o mass to your ship, but the kick will more than make up for it.' He scowls. 'Don't expect it to be pretty, though."

Reviewer: ```suggestion
	description `The engineer in front of you looks skeptically at your ship. "Don't worry, this is a piece of cake. I can have her accelerating ten percent faster with a few hours of work. It'll add a bit 'o mass to your ship, but the kick will more than make up for it." He scowls. "Don't expect it to be pretty, though."`
```
